src/components/container.js

```diff
@@ -0,0 +1,89 @@
+import React, { useState, useEffect } from "react";
+import Header from "./header";
+import TodoList from "./todo";
+import InputTodo from "./input";
+import { v4 as uuidv4 } from "uuid";
+
+const TodoContainer = () => {
+
+  const [todos, setTodos] =  useState(getIntitailTodos())
+
+  const handleChange = (id) => {
+    setTodos(prevState => 
+      prevState.map(todo => {
+        if (todo.id === id) {
+          return {
+            ...todo,
+            completed: !todo.completed
+          }
+        }
+
+        return todo
+      })
+    )
+  }
+
+  const delTodo = id => {
+    setTodos([
+        ...todos.filter(todo => {
+          return todo.id !==id
+        }),
+      ])
+  }
+
+  const addTodoItem = title => {
+    const newTodo = {
+      id: uuidv4(),
+      title: title,
+      completed: false
+    };
+
+    setTodos([...todos, newTodo])
+  }
+
+  const setUpdate = (updateTitle, id)  => {
+    setTodos(
+      todos.map(todo => {
+        if (todo.id === id) {
+          todo.title = updateTitle
+        }
+        return todo
+      }),
+    )
+  } 
+
+  // useEffect(() => {
+  //   console.log("test run")
+  //   const temp = localStorage.getItem("todos");
+  //   const loadedTodos = JSON.parse(temp)
+ //   if (loadedTodos) {
+  //     setTodos(loadedTodos)
+  //   }
+  // }, [setTodos])
```

Comment:
Comments in your codebase sometimes reduce the performance of your app and make the codebase less readable, kindly remove this part of the code